helpers/utils.mjs

```diff
@@ -29,7 +29,7 @@ export default async function getDeveloperWallet() {
   }
 
   console.log("\n");
-  return resultJson.wallet.ethAddress;
+  return resultJson.wallet.ethAddress || resultJson.wallet.solAddress;
```

Comment:
this seems very confusing. Make 2 functions instead `getDeveloperEvmAddress`, `getDeveloperSolAddress`

Reply:
Same for user functions below `getUserEvmAddress`, `getUserSolAddress`

Reply:
done

Reply:
We don't need to duplicate the whole function. 99% of the code is the same. Also, see the function names I mentioned above: they end in Address for more clarity.

Reply:
done